handlers/hierarchy.go

```diff
@@ -58,7 +58,7 @@ func (f *Filter) HierarchyUpdate(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
-	fil, err := f.FilterClient.GetJobState(req.Context(), userAccessToken, "", collectionID, "", filterID)
+	fil, err := f.FilterClient.GetJobState(req.Context(), userAccessToken, "", "", collectionID, filterID)
```

Comment:
I think this was a bug in the original code (we were switching collectionID and downloadServiceToken parameters according to GetJobState method signature in dp-api-clients-go)

Reply:
Probably was fine until we update the package here but yes I remember doing this as this seemed a more logical order